src/main/java/com/unorganized/UnorganizedController.java

```diff
@@ -59,17 +59,17 @@ public String test(Model model)
 	}
 	
 	@RequestMapping(value="/list", method=RequestMethod.GET)
-	public String read(Model model) 
+	public String list(Model model)
```

Comment:
The controller methods, mapped url, and returned view should have the same name, it helps with consistency. This way if you want to see the code for the list url, you can be sure that the view will be list.html and the method will be list(). Otherwise you will have to remember what url maps to what view and method.